tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -12,6 +12,20 @@ Do one of the following actions to access the Runtime Resources Visibility plug-
 - [Install the Tanzu Application Platform Full or View profile](../../install-intro.md)
 - [Install Tanzu Application Platform without using a profile](../../install-intro.md) and then
   install [Tanzu Application Platform GUI separately](../install-tap-gui.md)
+- Make sure to review the section [If you have a metrics server](#a-idmetrics-servera-if-you-have-a-metrics-server)
+
+## <a id="metrics-server"></a> If you have a metrics server
+
+By default, the Kubernetes API will be ignoring the metrics server on your cluster, if you want to access
+this information, you will need to go to your cluster section on the `app-config.yaml` and make sure
+the `skipMetricsLookup` attribute is set to `false`. 
+
+### Caution
```

Comment:
**question**: 

Is it intentional to use a subtitle instead other element e.g blockquote?
The code is a little different:

```
> **_Caution:_** If you set this value to `false` but do you not possess a metrics server running, you will get an error
as the follow:
![Screenshot of the RRV plugin error message due to metrics](images/metrics-server-error.png)
```

It looks like this:
<img width="666" alt="metrics-error" src="https://user-images.githubusercontent.com/89466662/186777752-491e088b-ab9e-41ba-b340-bcb2832d7063.png">